agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/diagnostics/DiagnosticsHelper.java

```diff
@@ -37,21 +37,21 @@ public static void initRpIntegration(Path agentPath) {
     // TODO (heya) how should we report functions windows users who are using app services
     //  windows attach by manually setting the env vars (which was the old documented way)
     if ("java".equals(System.getenv("FUNCTIONS_WORKER_RUNTIME"))) {
-      SdkVersionPrefixHolder.setRpIntegrationChar('f');
+      PropertyHelper.setRpIntegrationChar('f');
```

Comment:
we may be able to simplify now by using ResourceProvider.initResourceProvider() which is already doing the detection using the env vars (ok to follow-up on this after the release)

Reply:
will do that after. require a newer version of exporter if we want to move logic to ResourceProvider and reuse it everywhere.

Reply:
added a todo